api_keys.json

```
@@ -0,0 +1 @@
+{"0566dbe21ee33bba9419549716cd6f1f": {"rate_limit": 10, "usage": 0}, "e03da67439c0b7e7a622dde4fa3cf857": {"rate_limit": 10, "usage": 0}}
```

Make sure to remove this
```suggestion
{}
```